index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="main-wrapper">
+      <header class="toolbar">
+        <div class="logo-place"><h3 class="my-logo hex">My Logo</h3></div>
+        <figure class="icons">
+          <img src="images/Icons/Union.png" alt="Icons" />
+        </figure>
+      </header>
+      <section class="headline">
+        <h1>I’m Rushema Glad to see you!</h1>
+        <p class="text">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h2 class="hex connect">LET’S CONNECT</h2>
+        <div class="social-media">
+          <div clalss="socials">
+            <a href="#"><img src="images/socials/twitter.svg" /></a>
+          </div>
+          <div clalss="socials">
+            <a href="#"><img src="images/socials/Linkedin.svg" /></a>
+          </div>
+          <div clalss="socials">
+            <a href="#"><img src="images/socials/m.svg" /></a>
+          </div>
+          <div clalss="socials">
+            <a href="#"><img src="images/socials/Vector1.svg" /></a>
+          </div>
+          <div clalss="socials">
+            <a href="#"><img src="images/socials/Vector2.svg" /></a>
+          </div>
+        </div>
```

Comment:
Kindly update your list of social links to be wrapped in a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this
`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`